app/controllers/cori_questions_controller.rb

```diff
@@ -1,11 +1,17 @@
 class CoriQuestionsController < ApplicationController
   before_action :authenticate_user!
-  
+
   def index
     if params[:yes_or_no] == "no"
-      redirect_to "/pages/nocori"
+        redirect_to "/pages/nocori"
     elsif params[:yes_or_no] == "yes"
-      redirect_to new_sex_offender_path
+      respond_to do |format|
+        format.html {
+        redirect_to new_sex_offender_path }
+        format.json { render json: { action: "/sex_offenders", id: "sex-offender-question",
+        paragraph: "Have you been a registered sex offender at any point in the last 15 years?"}
```

Comment:
Line is too long. [96/80]<br>Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.